package.json

```
@@ -0,0 +1,6 @@
+{
+    "name": "demo",
```

We should add a description

```suggestion
    "name": "demo",
    "description": "Hello Worls Demo"
```